### PR TITLE
DEPLOY-401

### PR DESCRIPTION
**Pull request by gavincornwell**
Integrates the ACS image containing identity service support into the DBP and disables the RM and sync service AMPs as they are not compatible with 6.0 yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,16 +64,12 @@ kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
...
---set alfresco-content-services.repository.environment.SYNC_SERVICE_URI="http://$ELBADDRESS:$INFRAPORT/syncservice" \
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS:$INFRAPORT/auth" \
 --namespace=$DESIREDNAMESPACE
 
 #On AWS
 helm install alfresco-incubator/alfresco-dbp \
---set alfresco-sync-service.activemq.broker.host="${INFRARELEASE}-activemq-broker" \
---set alfresco-content-services.repository.environment.ACTIVEMQ_HOST="${INFRARELEASE}-activemq-broker" \
---set alfresco-content-services.repository.environment.SYNC_SERVICE_URI="http://$ELBADDRESS/syncservice" \
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS:$INFRAPORT/auth" \
```
Comment (svidrascu) [inline README.md]: You shouldnt have an infraport here, on aws.

**Comment by gavincornwell** [comment]
Good catch @svidrascu, README updated.

File: charts/incubator/alfresco-dbp/values.yaml [hunk 2]
```diff
@@ -51,4 +57,4 @@ alfresco-content-services:
         memory: "1500Mi"
 
 alfresco-sync-service:
-  enabled: true
\ No newline at end of file
+  enabled: false
```
Comment (rgauss) [inline charts/incubator/alfresco-dbp/values.yaml]: Does the sync service itself fail if the AMPs aren't applied?  If not, then perhaps we should leave it enabled just to continue to exercise as many of the components as possible.

Comment (gavincornwell) [reply charts/incubator/alfresco-dbp/values.yaml]: Yes, I’m pretty sure the pod failed to start or there were errors in the log which is what led me to disabling it but I’ll double check tomorrow.

Comment (gavincornwell) [reply charts/incubator/alfresco-dbp/values.yaml]: As I thought, an error during startup of the sync service:

```ERROR [2018-03-27 21:31:10.830] [main] org.alfresco.service.subscription.impl.config.ConfigurationImpl - Problem getting sync service configuration from repository: HTTP/1.1 404```

Sync Service needs the AMP installed as it provides webscripts it's expecting to find so disabling the sync service is correct unfortunately.

**Pull request merged**